src/modules/commandrunner/src/lib/CommandRunner.cpp

```diff
@@ -43,7 +46,7 @@ CommandRunner::CommandRunner(std::string clientName, unsigned int maxPayloadSize
 
 CommandRunner::~CommandRunner()
 {
-    OsConfigLogInfo(CommandRunnerLog::Get(), "CommandRunner %s shutting down", m_clientName.c_str());
+    OsConfigLogInfo(CommandRunnerLog::Get(), "CommandRunner '%s' shutting down", m_clientName.c_str());

```

Comment:
> "CommandRunner '%s' shutting down [](http://example.com/codeflow?start=44&length=34)

Please change this. The current way does not change anything. Add "session for" and think how we can diff between multiple sessions for the same client.

And instead of "shutting down" use "end" or "terminate"